en/docs/learn/querying-saml-assertions.md

```diff
@@ -32,29 +32,23 @@ authority.
 !!! note "Before you begin"
 
     1.  Clone or download the client application from [this GitHub
-        location](https://github.com/gayangithub/wso2-is-assertion-query-client)
-        and extract it. This folder location will be referred to as
+        location](https://github.com/wso2/samples-is),
+        extract it and navigate to
+        `           sample-saml-query-profile/saml-query-profile-client          `.
+        This folder location will be referred to as
         `            <CLIENT_HOME>           ` in this document.
 
     2.  Build the client application by executing the following command on
-        the `             <CLIENT_HOME>            ` root folder.
+        the `             <CLIENT_HOME>            ` folder.
 
         ``` java
         mvn clean install
         ```
 
 1.  Start the WSO2 Identity Server and log in to the management console.
```

Comment:
Sign in to WSO2 Identity Server Management Console.

Reply:
Fixed